pkg/types/baremetal/platform.go

```diff
@@ -30,6 +30,25 @@ type Platform struct {
 	// +optional
 	ClusterProvisioningIP string `json:"provisioningHostIP,omitempty"`
 
+	// ProvisioningInterface is the network interface used to provision new hosts.
+	// +optional
+	ProvisioningInterface string `json:"provisioningInterface"`
```

Comment:
if these are optional, please add `omitempty` for json tag. also provide details on the default when not explicitly specified.

Reply:
Sorry need to straighten all these out.  It's actually not optional even :).